message.c

```diff
@@ -626,12 +650,21 @@ parse_packet(const unsigned char *from, struct interface *ifp,
             debugf("Received nh %s (%d) from %s on %s.\n",
                    format_address(nh), message[2],
                    format_address(from), ifp->name);
-            if(message[2] == 1) {
-                memcpy(v4_nh, nh, 16);
-                have_v4_nh = 1;
-            } else {
-                memcpy(v6_nh, nh, 16);
-                have_v6_nh = 1;
+            switch(message[2]) {
+                case AE_IPV4:
```

Comment:
babeld style seems to put `case` statements at the same indent level than `switch` statements